src/mito2/src/sst/index.rs

```diff
@@ -278,6 +318,53 @@ impl<'a> IndexerBuilder<'a> {
 
         None
     }
+
+    fn build_bloom_filter_indexer(&self) -> Option<BloomFilterIndexer> {
+        let create = true; // TODO(zhongzc): add config for bloom filter
+
+        if !create {
+            debug!(
+                "Skip creating bloom filter due to config, region_id: {}, file_id: {}",
+                self.metadata.region_id, self.file_id,
+            );
+            return None;
+        }
+
+        let mem_limit = Some(100 * 1024 * 1024); // TODO(zhongzc): add config for bloom filter
+        let indexer = BloomFilterIndexer::new(
+            self.file_id,
+            self.metadata,
+            self.intermediate_manager.clone(),
+            mem_limit,
+        );
+
+        let err = match indexer {
+            Ok(indexer) => {
+                if indexer.is_none() {
+                    debug!(
+                        "Skip creating bloom filter due to no columns require indexing, region_id: {}, file_id: {}",
+                        self.metadata.region_id, self.file_id,
+                    );
+                }
+                return indexer;
+            }
+            Err(err) => err,
+        };
+
+        if cfg!(any(test, feature = "test")) {
+            panic!(
+                "Failed to create bloom filter, region_id: {}, file_id: {}, err: {}",
+                self.metadata.region_id, self.file_id, err
+            );
```

Comment:
non-blocking: Consider using debug format to print the error.
```suggestion
            panic!(
                "Failed to create bloom filter, region_id: {}, file_id: {}, err: {:?}",
                self.metadata.region_id, self.file_id, err
            );
```